auto_nag/scripts/workflow/no_severity.py

```diff
@@ -194,7 +194,19 @@ def get_bz_params(self, date):
                     "f18": "component",
                     "o18": "everchanged",
                     "f19": "CP",
-                    "f20": "CP",
+                    "j20": "OR",
+                    "f20": "OP",
+                    "f21": "bug_severity",
+                    "o21": "changedfrom",
+                    "v21": "critical",
+                    "f22": "bug_severity",
+                    "o22": "changedfrom",
+                    "v22": "major",
+                    "f23": "bug_severity",
+                    "o23": "changedfrom",
+                    "v23": "blocker",
```

Comment:
We could exclude these if severity was changed recently, but keep reporting about them if it was changed a long time ago. Not sure how easy it would be to implement though, so we can also just keep it this way.

Reply:
It is not hard to do; the downside is that the query is already long and it will be longer.

Reply:
All right, let's keep it as it is, we'll hopefully not need escalation for these bugs.